app/controllers/activitypub/followers_synchronizations_controller.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class ActivityPub::FollowersSynchronizationsController < ActivityPub::BaseController
+  include SignatureVerification
+  include AccountOwnedConcern
+
+  before_action :require_signature!
+  before_action :set_items
+  before_action :set_cache_headers
+
+  def show
+    expires_in 0, public: false
+    render json: collection_presenter, content_type: 'application/activity+json', serializer: ActivityPub::CollectionSerializer, adapter: ActivityPub::Adapter
+  end
+
+  private
+
+  def set_items
+    @items = @account.followers.where(domain: signed_request_account.domain).pluck(:uri).sort
```

Comment:
Might potentially want to add pagination logic since someone could have more than a thousand followers from a single domain even right now.

Reply:
Actually, there doesn't seem to be a good reason to use alphabetic sort here at all. When another server loads this data, it's not used to build a digest, individual items are used and order does not matter. So, using standard pagination order would ensure that the collection is deterministically paginable.